src/views/ManageList.jsx

```diff
@@ -1,7 +1,38 @@
+import { addItem } from '../api/firebase';
+import toast, { Toaster } from 'react-hot-toast';
+
 export function ManageList() {
+	const notify = () => toast.success('Item added to list!');
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		addItem({ itemName: item });
+		console.log('hi');
+		notify();
+	};
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<div>
+			<p>
+				Hello from the <code>/manage-list</code> page!
+			</p>
+			<form onSubmit={handleSubmit}>
```

Comment:
hey @RossaMania what do you think about using `react-hook-form` on our form? I know we got the base going and I think all in the form we have left to do it capture the state of the form to pass the correct parameters to the addItem. 

We can make sure validation is in the form either way (make sure they put in an item name and select an time option) but this may make it a bit quicker as we make future iterations, but I also could be jumping the gun. :sweat_smile:

https://react-hook-form.com/form-builder

https://www.npmjs.com/package/react-hook-form

Reply:
While I think it could still be an option as we add functionality I rescind my suggestion. Lol I don't think it is needed right now and will maybe cause some scope creep.